src/styles/app/32-command-palette.css

```
@@ -101,6 +101,51 @@
+}
+
+.commandSearchSaveButton:active:not(:disabled) {
+	scale: 0.96;
+}
+
+.commandSearchSaveButton[data-saved="true"] {
+	color: var(--text-accent);
+}
+
+.commandSearchSaveButton[data-saved="true"] svg path {
+	fill: currentColor;
```

Stylelint (17.14.0)

[error] Expected "currentColor" to be "currentcolor" (value-keyword-case)

(value-keyword-case)